### PR TITLE
feat: added an way to not have your personal data publically leaked when you are an orga

### PR DESCRIPTION
**Pull request by TeddyRoncin**
# Opt-in for data publically revealed

## Changes

* modified route GET /users/orgas
* added route PATCH /admin/users/trombi
* added table Orga to store information about organizers
  - store orga preferences
  - store the file the photo is saved at

## Breaking changes

everything using trombi data

### CODE REVIEW

**Review by DevNono — APPROVED** [review]

File: src/controllers/admin/openapi.yml [hunk 1]
```diff
@@ -1302,6 +1307,43 @@
       409:
         $ref: '#/components/responses/409PlaceAlreadyAttributed'
 
+/admin/users/trombi:
+  patch:
+    summary: Modifie les préférences de confidentialité d'un organisateur et renvoie le nom d'un fichier dans lequel il peut stocker sa photo.
+    description: |-
+      L'organisateur peut opt-in pour afficher son nom et son prénom, ainsi que pour montrer sa photo de profil.
+      Un nom de fichier est renvoyé, qui permet à l'organisateur de stocker son nom de fichier dans
+      "<uploads url>/orgas/<nom du fichier>.png".
```
Comment (DevNono) [inline src/controllers/admin/openapi.yml:1316]: webp

File: tests/admin/upload/deleteFile.test.ts [hunk 2]
```diff
@@ -16,7 +17,9 @@ describe('DELETE /admin/upload', () => {
   const path = encodeURIComponent('tournaments/lol-logo.webp');
 
   after(async () => {
+    await database.orga.deleteMany();
     await database.user.deleteMany();
+    uploads.existingFiles.push('tournaments/lol-logo.png');
```
Comment (DevNono) [inline tests/admin/upload/deleteFile.test.ts:22]: webp

File: src/utils/filters.ts [hunk 3]
```diff
@@ -34,7 +34,6 @@ export const filterUser = (user: User) =>
     'attendant.firstname',
     'attendant.lastname',
     'attendant.id',
-    'orgaRoles',
```
Comment (DevNono) [inline src/utils/filters.ts:37]: On peut plus filter par roles ?

**Review by DevNono — CHANGES_REQUESTED** [review]

File: src/utils/filters.ts (hunk 3 above)
Comment (TeddyRoncin) [reply src/utils/filters.ts:37]: heu non, on dit juste qu'on ne retourne pas orgaRoles, qui n'existe plus maintenant

**Review by DevNono — APPROVED** [review]

**Pull request merged**